src/lib/features/project/project-owners-read-model.test.ts

```diff
@@ -20,7 +20,7 @@ const mockProjectData = (name: string): ProjectForUi => ({
     createdAt: new Date(),
     favorite: false,
     lastReportedFlagUsage: null,
-    lastFlagUpdate: null,
+    lastUpdatedAt: null,
```

Comment:
I've talked to stakeholder Jaanus about this, and we might want to include project updates here in the future, so we'll use a name that's less specifically about flags. Additionally: this name is still in development anyway, so it might change in the future.